wpeverest/user-registration
# Tweak - Addons JSON pinged from local

### All Submissions:






### Changes proposed in this Pull Request:
Previously we were pinging remotely every time the user visits the addons page of our plugin, the images, contents, and preview links were fetched from that URL. Now in this PR, those assets are fetched locally.

### How to test the changes in this Pull Request:

1. Visit the addons page and click on the Refresh Addons button and verify if all the images, contents, and preview link of addons are intact and working as expected.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Addons JSON pinged from local.
